lib/bandit/http1/adapter.ex

```diff
@@ -335,20 +335,26 @@ defmodule Bandit.HTTP1.Adapter do
   response_reasons = %{
     100 => "Continue",
     101 => "Switching Protocols",
+    102 => "Processing",
+    103 => "Early Hints",
```

Comment:
Not in 9110

Reply:
[RFC 8297](https://httpwg.org/specs/rfc8297.html)

It would be odd to serve early hints from Bandit apps that serve HTML and not have this text included.